### PR TITLE
data separation added, hand recogniztion error fixed

### PR DESCRIPTION
**Pull request by sangbin-sangbin**

### CODE REVIEW

**Comment by kmsgm** [comment]
Please remove `__pycache__`. How about add `__pycache__` in `.gitignore` file?

File: run.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 
 def run(hand_tracker, model):
-    gestures = ["default", "left", "right", "select", "exit", "none"]
+    gestures = ["default", "left", "right", "select", "exit", "shortcut1", "shortcut2", "none"]
```
Comment (kmsgm) [inline run.py:12]: Plz edit gestures in `HandTracker.py` file as well.

**Comment by sangbin-sangbin** [comment]
> Please remove `__pycache__`. How about adding `__pycache__` in `.gitignore` file?

fixed

File: dataset_checker.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    model = Model(INPUT_SIZE, HIDDEN_DIM1, HIDDEN_DIM2, TARGET_SIZE)
+    loss_function = nn.CrossEntropyLoss()
+    optimizer = optim.SGD(model.parameters(), lr=0.01)
+
+    total_epochs = 10
+    total_steps = len(train_dataset)
+
+    prev_loss = float("inf")
+    tolerance = 20
```
Comment (kmsgm) [inline dataset_checker.py:68]: total_epochs is 10. Could tolerance be 20?

File: test_utils.py [hunk 3]
```diff
@@ -30,6 +30,8 @@ def same_hand_tracking(hands, prev_pos, same_hand_threshold):
             min_idx = i
             min_val = distance
 
+    print(min_val)
```
Comment (kmsgm) [inline test_utils.py:33]: Is this print necessary?

File: test.py [hunk 4]
```diff
@@ -8,26 +8,29 @@
...
-    HIDDEN_DIM = 32
-    TARGET_SIZE = 6
+    HIDDEN_DIM1 = 32
+    HIDDEN_DIM2 = 32
+    TARGET_SIZE = 8
 
-    model = Model(INPUT_SIZE, HIDDEN_DIM, TARGET_SIZE)
-    model.load_state_dict(torch.load("model2.pt"))
+    model = Model(INPUT_SIZE, HIDDEN_DIM1, HIDDEN_DIM2, TARGET_SIZE)
+    model.load_state_dict(torch.load("model.pt"))
```
Comment (kmsgm) [inline test.py:33]: if `model2.pt` is not used, please remove this

File: dataset_checker.py (hunk 2 above)
Comment (sangbin-sangbin) [reply dataset_checker.py:68]: I think we have to figure out better way to early stop than tolerance

File: test_utils.py (hunk 3 above)
Comment (sangbin-sangbin) [reply test_utils.py:33]: no fixed

File: test.py (hunk 4 above)
Comment (sangbin-sangbin) [reply test.py:33]: I removed

File: run.py (hunk 1 above)
Comment (sangbin-sangbin) [reply run.py:12]: edited

**Comment by kmsgm** [comment]
LGTM

**Pull request merged**